source/docs/software/wpilib-tools/robot-characterization/characterization-routine.rst

```diff
@@ -53,6 +53,15 @@ Follow the instructions in the pop-up windows after pressing each test button.
 .. image:: images/running-the-test.png
    :alt: Running the various tests
 
+Trackwidth
+^^^^^^^^^^
+
+The Trackwidth test will spin your robot in an attempt to measure the distance between your wheels.
+
+.. note:: If your robot is having trouble turning during the Trackwidth test you should increase the :guilabel:`Rotation Wheel voltage (V):` value and run the test again.
+
+.. note:: Trackwidth values for high friction wheels (like pneumatic tires) may be measured incorrectly and a measured value should be used.
```

Comment:
Might not be good as is, but here's a start:
```suggestion
The trackwidth test will spin your robot to determine an empirical trackwidth. It compares how far the wheel encoders drove against the reported rotation from the gyroscope. If the encoder distances are small relative to the change in heading, the empirical trackwidth will be smaller than the real value to reflect wheel scrub. A common cause for this would be high-friction wheels (e.g., pneumatic tires).

This empirical value should be used everywhere in code instead of the real value measured with a tape measure.

.. note:: If your robot is having trouble turning during the Trackwidth test you should increase the :guilabel:`Rotation Wheel voltage (V):` value and run the test again.
```

Reply:
From your first paragraph it sounds like the trackwidth would be smaller.  In practice I observed a trackwidth >2X actual.